src/test/mochitest/head.js

```diff
@@ -679,9 +688,9 @@ function togglePauseOnExceptions(
  * @return {Promise}
  * @static
  */
-function invokeInTab(fnc) {
+async function invokeInTab(fnc) {
   info(`Invoking function ${fnc} in tab`);
-  return ContentTask.spawn(gBrowser.selectedBrowser, fnc, function*(fnc) {
+  ContentTask.spawn(gBrowser.selectedBrowser, fnc, function*(fnc) {
```

Comment:
Won't failing to return the promise here break `uncaughtException` and `caughtException` in `src/test/mochitest/browser_dbg-pause-exceptions.js`?

Reply:
we aren't using the promise, but i shouldn't have removed it either